controllers/apps/v1beta3/emqx_test.go

```diff
@@ -419,7 +420,55 @@ func check_statefulset(emqx v1beta3.Emqx) {
 		Expect(sts.Spec.VolumeClaimTemplates).Should(ContainElements(HaveField("ObjectMeta", HaveField("Name", names.Data()))))
 		Expect(sts.Spec.VolumeClaimTemplates).Should(ContainElements(HaveField("ObjectMeta", HaveField("Name", names.Log()))))
 	}
+	//Env
+
```

Comment:
Delete empty lines
What's different for BrokerEnvVars and EnterpriseEnvVars ?

Reply:
just like EMQX_CLUSTER__DNS__APP , in broker the value is emqx, if in enterprise, the value is emqx-ee

Reply:
Maybe `{Name: "EMQX_CLUSTER__DNS__NAME", Value: fmt.Sprintf("%s-headless.%s.svc.cluster.local", emqx.GetName(), emqx.GetNamespace()) },` is work